nemo_curator/modules/distributed_data_classifier.py

```diff
@@ -324,37 +352,39 @@ class QualityClassifier(DistributedDataClassifier):
     def __init__(
         self,
         model_path,
-        labels,
+        num_labels=3,
         filter_by=None,
         batch_size=256,
-        out_dim=None,
         pred_column="quality_pred",
         prob_column="quality_prob",
         max_chars=6000,
         device_type="cuda",
         autocast=True,
     ):
-        if len(labels) == 2:
-            out_dim = 1  # Binary classification
+        if num_labels == 3:
+            self.labels = ["High", "Medium", "Low"]
+            self.out_dim = num_labels  # Multiclass classification
+        elif num_labels == 2:
+            self.labels = ["Medium_High", "Low"]
+            self.out_dim = 1  # Binary classification
```

Comment:
We can use the logic from https://github.com/NVIDIA/NeMo-Curator/pull/122 for the quality classifier for now, until we release the quality model on HuggingFace.

Reply:
Makes sense.